.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,26 @@
 ### Summary
-Summarize the contents of the code changes in your pull request. Tag any open issues you believe to be resolved by this pull request.
+<!--- Summarize and explain the reason behind these code changes. What are the changes, and why are they necessary? -->
+
+
+<!--- Include any issue addressed by this pull request. -->
+<!--- Example: Closes #45 -->
+Closes #
+
+### Deployment Link
+<!---Include the deployment link, if applicable. -->
+<!--- Example: https://terra-core-deployed-pr-45.herokuapp.com/ -->
+https://terra-core-deployed-pr-#.herokuapp.com/
+
+### Testing
+<!-- Demonstrate that these changes are stable. How have these changes been verified? -->
 
 ### Additional Details
-If you have anything else that you think may be relevant to this issue, list it here. Additional information can help us better understand your changes and speed up the review process.
+<!-- List anything else that is relevant to this issue. Additional information will help us better understand your changes and speed up the review process. -->
 
+### Contributing
 Please add your name to the [CONTRIBUTORS.md] file. Adding your name to the [CONTRIBUTORS.md] file signifies agreement to all rights and reservations provided by the [License].
 
```

Comment:
Can we add a comment for this section? I always delete it

Reply:
https://github.com/cerner/terra-core/pull/2912/commits/0efcf1147eb12adf4701321449afc8d9b57bb447